refunder_logger.rb

```diff
@@ -0,0 +1,43 @@
+module Spree
+  module Retail
+    module Shopify
+      class RefunderLogger
```

Comment:
I would abstract stuff into a General Logger that specific logger would inherit of. 
That way the specific logger would be more explicit in what it does.